src/main/java/WayofTime/bloodmagic/client/render/RenderMasterRitualStone.java

```diff
@@ -0,0 +1,95 @@
+package WayofTime.bloodmagic.client.render;
+
+import WayofTime.bloodmagic.api.ritual.Ritual;
+import WayofTime.bloodmagic.api.ritual.RitualComponent;
+import WayofTime.bloodmagic.tile.TileMasterRitualStone;
+import WayofTime.bloodmagic.util.handler.event.ClientHandler;
+import net.minecraft.client.Minecraft;
+import net.minecraft.client.entity.EntityPlayerSP;
+import net.minecraft.client.renderer.GlStateManager;
+import net.minecraft.client.renderer.texture.TextureAtlasSprite;
+import net.minecraft.client.renderer.tileentity.TileEntitySpecialRenderer;
+import net.minecraft.util.EnumFacing;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.World;
+import org.lwjgl.opengl.GL11;
+
+public class RenderMasterRitualStone extends TileEntitySpecialRenderer<TileMasterRitualStone>
```

Comment:
The same thing you used for the ritual rendering before? The data doesn't need to be saved serverside and/or synced so I'm actually not sure why you're using fields in the Tile's themselves either.